Docs/sphinx_documentation/source/External_Profiling_Tools.rst

```diff
@@ -279,3 +279,167 @@ generated figures. Some examples are shown here.
 
 .. [5]
    https://www.nersc.gov/users/software/performance-and-debugging-tools/ipm/
+
+NSight Systems
+==============
+
+The NSight Systems tool provides a high-level overview of a GPU code, displaying the kernel
```

Comment:
Nsight Systems can also be used for CPU-only code, it's a useful tool (when combined with NVTX) if you want to see on your graphical timeline where the time is spent in the CPU code so you know what things to prioritize for acceleration. I'd prefer if that were at least mentioned here.